README.md

```diff
@@ -27,7 +27,7 @@ sbr-control-api is a Play Framework application written predominantly in Scala.
 
 To install SBT quickly you can use Homebrew ([Brew](http://brew.sh)):
 ```shell
-brew install sbt
+brew install gradle
```

Comment:
Those not on a mac should check out [sdkman](http://sdkman.io/)

Of course, the wrapper has been checked into the project.  Perhaps we should explain that a local install of gradle is not required, and show some examples of gradlew usage for those that prefer that approach ...

Reply:
This is a good point. I have configured the wrapper to work with artifactory before, but it will be a pain adding the switch for on/off network. Maybe the wrapper is only adding complexity here.